src/components/VmDetails/cards/UtilizationCard.js

```diff
@@ -20,160 +23,333 @@ import {
   SparklineChart,
 } from 'patternfly-react'
 
+const CpuCharts = ({ stats, isRunning }) => {
```

Comment:
Could you, please, split this file. Because it has to much components. And also it would be nice if every card would have own directory.

Reply:
> Could you, please, split this file. Because it has to much components.

The entire file with 7 component is <400 lines so it doesn't really both me.  There are other places where there is a collection of components in a single file or smaller components in 1 file with bigger components in another file.  So there isn't a consistent standard across the app.

> And also it would be nice if every card would have own directory.

That does make sense.  With your snapshots in a directory and the `UtilizationCard` in a directory (broken apart maybe), that would be self-consistent.  I'll split up `UtilizationCard` and if it still looks ok, I'll post an update.